src/constants/style.scss

```diff
@@ -26,6 +26,9 @@ $tooltip-background-color--light: #d3daf0;
 $tooltip-background-color--dark: #485064;
 $color-progress-circle: #3c7fff;
 $color-icon-light-blue: #9ebfff;
+$color-board-reaction-name--dark: #5a647d;
+$color-board-reaction-name-self--light: #0057ff;
+$color-board-reaction-name-self--dark: #6096ff;
```

Comment:
why not just use the colors we already have?

Reply:
i just followed the design here 🤷 maybe @SelinaBuff could take a look?

Reply:
the colors are correct, I'll keep them as is, they just have to be renamed when #3267 is done.